specification/communication/data-plane/Identity/preview/2025-03-02-preview/examples/CreateOrReplaceTeamsPhoneAssignment.json

```
@@ -0,0 +1,28 @@
+{
+  "parameters": {
+    "api-version": "2025-03-02-preview",
+    "content-type": "application/json",
+    "tenantId": "c70d068c-2f32-484d-9455-73070cd5aadd",
+    "objectId": "b27b64b9-39d2-44a3-869d-67700b74e805",
+    "body": {
+      "principalType": "teamsResourceAccount"
```

```suggestion
      "principalType": "resourceAccount"
```

also in other places